### PR TITLE
feat(eslint): script to report on top problems

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Add a script to count the number of issues for individual ESLint rules

## Why?

So we can keep track of what’s happened. This can only be run locally as a helper.

`deno run --A scripts/deno/eslint.ts` with optional flag `--lint` to run ESLint

Inspired by the work on #4935 

--- 

## Example

<details>
<summary><code>@typescript-eslint/explicit-module-boundary-types</code> : <strong>718</strong> problems over 330 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/Ad.tsx</li>
<li>./dotcom-rendering/src/amp/components/Caption.tsx</li>
<li>./dotcom-rendering/src/amp/components/ContentABTest.tsx</li>
<li>./dotcom-rendering/src/amp/components/OnwardContainer.tsx</li>
<li>./dotcom-rendering/src/amp/components/Pagination.stories.tsx</li>
<li>./dotcom-rendering/src/amp/components/RegionalAd.tsx</li>
<li>./dotcom-rendering/src/amp/components/StickyAd.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/TwitterBlockComponent.tsx</li>
<li>./dotcom-rendering/src/amp/components/moustache.tsx</li>
<li>./dotcom-rendering/src/amp/components/topMeta/Byline.stories.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>import/no-extraneous-dependencies</code> : <strong>683</strong> problems over 345 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/AMPAttributes.ts</li>
<li>./dotcom-rendering/src/amp/components/AdConsent.tsx</li>
<li>./dotcom-rendering/src/amp/components/AmpExperiment.tsx</li>
<li>./dotcom-rendering/src/amp/components/Analytics.tsx</li>
<li>./dotcom-rendering/src/amp/components/AnalyticsIframe.tsx</li>
<li>./dotcom-rendering/src/amp/components/Blocks.tsx</li>
<li>./dotcom-rendering/src/amp/components/BodyArticle.tsx</li>
<li>./dotcom-rendering/src/amp/components/BodyLiveblog.tsx</li>
<li>./dotcom-rendering/src/amp/components/Caption.tsx</li>
<li>./dotcom-rendering/src/amp/components/ContentABTest.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/prefer-nullish-coalescing</code> : <strong>180</strong> problems over 70 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/Blocks.tsx</li>
<li>./dotcom-rendering/src/amp/components/BodyArticle.tsx</li>
<li>./dotcom-rendering/src/amp/components/Caption.tsx</li>
<li>./dotcom-rendering/src/amp/components/KeyEvents.tsx</li>
<li>./dotcom-rendering/src/amp/components/RegionalAd.test.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/ImageBlockComponent.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/TimelineBlockComponent.tsx</li>
<li>./dotcom-rendering/src/amp/lib/get-video-id.ts</li>
<li>./dotcom-rendering/src/amp/lib/image-fit.ts</li>
<li>./dotcom-rendering/src/amp/server/index.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/no-explicit-any</code> : <strong>121</strong> problems over 52 files</summary>
<ul>
<li>./dotcom-rendering/index.d.ts</li>
<li>./dotcom-rendering/src/amp/components/JsonScript.tsx</li>
<li>./dotcom-rendering/src/amp/components/MainMedia.tsx</li>
<li>./dotcom-rendering/src/amp/components/Sidebar.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/YoutubeBlockComponentAMP.tsx</li>
<li>./dotcom-rendering/src/amp/components/moustache.tsx</li>
<li>./dotcom-rendering/src/amp/lib/find-adslots.ts</li>
<li>./dotcom-rendering/src/amp/lib/permutive.ts</li>
<li>./dotcom-rendering/src/amp/server/document.tsx</li>
<li>./dotcom-rendering/src/lib/querystring.ts</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/no-unnecessary-condition</code> : <strong>112</strong> problems over 55 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/KeyEvents.tsx</li>
<li>./dotcom-rendering/src/amp/components/MainMedia.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/ImageBlockComponent.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/YoutubeBlockComponentAMP.tsx</li>
<li>./dotcom-rendering/src/amp/lib/ad-json.ts</li>
<li>./dotcom-rendering/src/amp/lib/image-fit.test.ts</li>
<li>./dotcom-rendering/src/lib/byline.ts</li>
<li>./dotcom-rendering/src/model/enhance-dividers.ts</li>
<li>./dotcom-rendering/src/model/enhance-images.ts</li>
<li>./dotcom-rendering/src/model/enhance-numbered-lists.ts</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/no-unsafe-assignment</code> : <strong>89</strong> problems over 36 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/MainMedia.tsx</li>
<li>./dotcom-rendering/src/amp/components/RegionalAd.test.tsx</li>
<li>./dotcom-rendering/src/amp/components/ShareIcons.tsx</li>
<li>./dotcom-rendering/src/amp/lib/enhance.ts</li>
<li>./dotcom-rendering/src/amp/lib/permutive.ts</li>
<li>./dotcom-rendering/src/amp/server/ampExperimentCache.ts</li>
<li>./dotcom-rendering/src/lib/assets.ts</li>
<li>./dotcom-rendering/src/lib/querystring.ts</li>
<li>./dotcom-rendering/src/model/article-sections.ts</li>
<li>./dotcom-rendering/src/model/extract-nav.ts</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/restrict-template-expressions</code> : <strong>66</strong> problems over 38 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/Analytics.tsx</li>
<li>./dotcom-rendering/src/amp/components/Footer.tsx</li>
<li>./dotcom-rendering/src/amp/components/OnwardContainer.tsx</li>
<li>./dotcom-rendering/src/amp/server/ampExperimentCache.ts</li>
<li>./dotcom-rendering/src/amp/server/document.tsx</li>
<li>./dotcom-rendering/src/amp/server/index.tsx</li>
<li>./dotcom-rendering/src/lib/fetchTickerData.ts</li>
<li>./dotcom-rendering/src/model/extract-nav.ts</li>
<li>./dotcom-rendering/src/model/validate.ts</li>
<li>./dotcom-rendering/src/server/prod-server.ts</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>consistent-return</code> : <strong>50</strong> problems over 25 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/elements/YoutubeBlockComponentAMP.tsx</li>
<li>./dotcom-rendering/src/server/dev-server.ts</li>
<li>./dotcom-rendering/src/server/prod-server.ts</li>
<li>./dotcom-rendering/src/web/browser/bootCmp/init.ts</li>
<li>./dotcom-rendering/src/web/components/ArticleMeta.tsx</li>
<li>./dotcom-rendering/src/web/components/CalloutBlockComponent.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/Card/components/ContentWrapper.tsx</li>
<li>./dotcom-rendering/src/web/components/Card/components/ImageWrapper.tsx</li>
<li>./dotcom-rendering/src/web/components/CardHeadline.tsx</li>
<li>./dotcom-rendering/src/web/components/Carousel.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>default-case</code> : <strong>50</strong> problems over 17 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/elements/YoutubeBlockComponentAMP.tsx</li>
<li>./dotcom-rendering/src/web/browser/relativeTime/updateTimeElements.ts</li>
<li>./dotcom-rendering/src/web/components/ArticleMeta.tsx</li>
<li>./dotcom-rendering/src/web/components/Card/components/ContentWrapper.tsx</li>
<li>./dotcom-rendering/src/web/components/Card/components/ImageWrapper.tsx</li>
<li>./dotcom-rendering/src/web/components/CardHeadline.tsx</li>
<li>./dotcom-rendering/src/web/components/ClickToView.tsx</li>
<li>./dotcom-rendering/src/web/components/ImageComponent.tsx</li>
<li>./dotcom-rendering/src/web/components/InteractiveBlockComponent.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/Lineup.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>react/no-danger</code> : <strong>48</strong> problems over 37 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/AmpExperiment.tsx</li>
<li>./dotcom-rendering/src/amp/components/Analytics.tsx</li>
<li>./dotcom-rendering/src/amp/components/Caption.tsx</li>
<li>./dotcom-rendering/src/amp/components/Elements.tsx</li>
<li>./dotcom-rendering/src/amp/components/JsonScript.tsx</li>
<li>./dotcom-rendering/src/amp/components/Sidebar.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/CommentBlockComponent.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/DisclaimerBlockComponent.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/ImageBlockComponent.tsx</li>
<li>./dotcom-rendering/src/amp/components/elements/PullquoteBlockComponent.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>no-underscore-dangle</code> : <strong>45</strong> problems over 18 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/Elements.tsx</li>
<li>./dotcom-rendering/src/amp/components/MainMedia.tsx</li>
<li>./dotcom-rendering/src/amp/lib/enhance.ts</li>
<li>./dotcom-rendering/src/amp/lib/find-adslots.ts</li>
<li>./dotcom-rendering/src/amp/lib/scripts.ts</li>
<li>./dotcom-rendering/src/model/enhance-blockquotes.ts</li>
<li>./dotcom-rendering/src/model/enhance-dividers.ts</li>
<li>./dotcom-rendering/src/model/enhance-dots.ts</li>
<li>./dotcom-rendering/src/model/enhance-embeds.ts</li>
<li>./dotcom-rendering/src/model/enhance-images.ts</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/prefer-optional-chain</code> : <strong>44</strong> problems over 33 files</summary>
<ul>
<li>./dotcom-rendering/src/model/enhance-images.ts</li>
<li>./dotcom-rendering/src/model/extract-ga.ts</li>
<li>./dotcom-rendering/src/model/find-pillar.ts</li>
<li>./dotcom-rendering/src/web/browser/bootCmp/init.ts</li>
<li>./dotcom-rendering/src/web/browser/initDiscussion/init.ts</li>
<li>./dotcom-rendering/src/web/browser/initPerf.ts</li>
<li>./dotcom-rendering/src/web/browser/newsletterEmbedIframe/init.ts</li>
<li>./dotcom-rendering/src/web/components/Callout/FileUpload.tsx</li>
<li>./dotcom-rendering/src/web/components/Callout/Select.tsx</li>
<li>./dotcom-rendering/src/web/components/CalloutBlockComponent.importable.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>no-console</code> : <strong>42</strong> problems over 31 files</summary>
<ul>
<li>./dotcom-rendering/scripts/jest/setup.ts</li>
<li>./dotcom-rendering/src/amp/components/Elements.tsx</li>
<li>./dotcom-rendering/src/amp/server/ampExperimentCache.ts</li>
<li>./dotcom-rendering/src/amp/server/document.test.tsx</li>
<li>./dotcom-rendering/src/server/lib/aws/aws-metrics.ts</li>
<li>./dotcom-rendering/src/server/lib/logging.ts</li>
<li>./dotcom-rendering/src/server/prod-server.ts</li>
<li>./dotcom-rendering/src/web/browser/bootCmp/init.ts</li>
<li>./dotcom-rendering/src/web/browser/islands/doHydration.ts</li>
<li>./dotcom-rendering/src/web/browser/islands/getName.ts</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>import/order</code> : <strong>40</strong> problems over 9 files</summary>
<ul>
<li>./dotcom-rendering/src/web/components/SignInGate/signInGate.ts</li>
<li>./dotcom-rendering/src/web/layouts/CommentLayout.tsx</li>
<li>./dotcom-rendering/src/web/layouts/ImmersiveLayout.tsx</li>
<li>./dotcom-rendering/src/web/layouts/InteractiveLayout.tsx</li>
<li>./dotcom-rendering/src/web/layouts/LiveLayout.tsx</li>
<li>./dotcom-rendering/src/web/layouts/ShowcaseLayout.tsx</li>
<li>./dotcom-rendering/src/web/layouts/StandardLayout.tsx</li>
<li>./dotcom-rendering/src/web/lib/decidePalette.ts</li>
<li>./dotcom-rendering/src/web/lib/renderElement.tsx</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/no-unsafe-argument</code> : <strong>39</strong> problems over 20 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/lib/find-adslots.test.ts</li>
<li>./dotcom-rendering/src/amp/server/index.tsx</li>
<li>./dotcom-rendering/src/lib/querystring.ts</li>
<li>./dotcom-rendering/src/model/extract-nav.ts</li>
<li>./dotcom-rendering/src/model/validate.test.ts</li>
<li>./dotcom-rendering/src/web/browser/islands/doHydration.ts</li>
<li>./dotcom-rendering/src/web/browser/newsletterEmbedIframe/init.ts</li>
<li>./dotcom-rendering/src/web/browser/sentryLoader/init.ts</li>
<li>./dotcom-rendering/src/web/browser/updateIframeHeight.tsx</li>
<li>./dotcom-rendering/src/web/components/CalloutBlockComponent.importable.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>eslint-comments/require-description</code> : <strong>37</strong> problems over 25 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/AMPAttributes.ts</li>
<li>./dotcom-rendering/src/amp/server/document.tsx</li>
<li>./dotcom-rendering/src/lib/pillars.ts</li>
<li>./dotcom-rendering/src/server/lib/aws/aws-metrics.ts</li>
<li>./dotcom-rendering/src/server/prod-server.ts</li>
<li>./dotcom-rendering/src/web/browser/dynamicImport/init.ts</li>
<li>./dotcom-rendering/src/web/browser/islands/doHydration.ts</li>
<li>./dotcom-rendering/src/web/browser/newsletterEmbedIframe/init.ts</li>
<li>./dotcom-rendering/src/web/browser/updateIframeHeight.tsx</li>
<li>./dotcom-rendering/src/web/components/Callout/Form.test.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>react/no-array-index-key</code> : <strong>34</strong> problems over 19 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/Elements.tsx</li>
<li>./dotcom-rendering/src/amp/components/Footer.tsx</li>
<li>./dotcom-rendering/src/amp/components/StarRating.tsx</li>
<li>./dotcom-rendering/src/amp/components/topMeta/TopMetaAnalysis.tsx</li>
<li>./dotcom-rendering/src/amp/components/topMeta/TopMetaLiveblog.tsx</li>
<li>./dotcom-rendering/src/amp/components/topMeta/TopMetaNews.tsx</li>
<li>./dotcom-rendering/src/amp/components/topMeta/TopMetaOpinion.tsx</li>
<li>./dotcom-rendering/src/amp/components/topMeta/TopMetaPaidContent.tsx</li>
<li>./dotcom-rendering/src/web/components/Callout/CheckboxSelect.tsx</li>
<li>./dotcom-rendering/src/web/components/Callout/Form.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/no-unsafe-call</code> : <strong>27</strong> problems over 12 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/lib/enhance.ts</li>
<li>./dotcom-rendering/src/lib/fonts-css.ts</li>
<li>./dotcom-rendering/src/web/browser/dynamicImport/init.ts</li>
<li>./dotcom-rendering/src/web/browser/islands/doHydration.ts</li>
<li>./dotcom-rendering/src/web/components/ElementContainer.tsx</li>
<li>./dotcom-rendering/src/web/components/MostViewedFooter.test.tsx</li>
<li>./dotcom-rendering/src/web/components/MostViewedRight.test.tsx</li>
<li>./dotcom-rendering/src/web/components/Onwards.test.tsx</li>
<li>./dotcom-rendering/src/web/components/ReaderRevenueLinks.test.tsx</li>
<li>./dotcom-rendering/src/web/components/RewrappedComponent.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>react/button-has-type</code> : <strong>23</strong> problems over 12 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/components/BodyLiveblog.tsx</li>
<li>./dotcom-rendering/src/amp/components/Epic.tsx</li>
<li>./dotcom-rendering/src/amp/components/Expandable.tsx</li>
<li>./dotcom-rendering/src/amp/components/Header.tsx</li>
<li>./dotcom-rendering/src/amp/components/topMeta/PaidForBand.tsx</li>
<li>./dotcom-rendering/src/web/components/Carousel.tsx</li>
<li>./dotcom-rendering/src/web/components/Dropdown.tsx</li>
<li>./dotcom-rendering/src/web/components/InteractiveContentsBlockComponent.tsx</li>
<li>./dotcom-rendering/src/web/components/MostViewedFooterGrid.tsx</li>
<li>./dotcom-rendering/src/web/components/SignInGate/gateDesigns/SignInGateFakeSocial.tsx</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/require-await</code> : <strong>22</strong> problems over 9 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/lib/image-fit.test.ts</li>
<li>./dotcom-rendering/src/server/dev-server.ts</li>
<li>./dotcom-rendering/src/server/lib/get-content-from-url.test.ts</li>
<li>./dotcom-rendering/src/server/prod-server.ts</li>
<li>./dotcom-rendering/src/web/browser/relativeTime/updateTimeElements.test.tsx</li>
<li>./dotcom-rendering/src/web/browser/sentryLoader/init.ts</li>
<li>./dotcom-rendering/src/web/components/MostViewedFooter.test.tsx</li>
<li>./dotcom-rendering/src/web/components/MostViewedRight.test.tsx</li>
<li>./dotcom-rendering/src/web/components/ReaderRevenueLinks.test.tsx</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/no-unsafe-return</code> : <strong>16</strong> problems over 15 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/lib/enhance.ts</li>
<li>./dotcom-rendering/src/lib/fonts-css.ts</li>
<li>./dotcom-rendering/src/model/article-sections.ts</li>
<li>./dotcom-rendering/src/model/extract-nav.ts</li>
<li>./dotcom-rendering/src/server/lib/aws/aws-parameters.ts</li>
<li>./dotcom-rendering/src/web/components/ArticleHeadline.tsx</li>
<li>./dotcom-rendering/src/web/components/ElementContainer.tsx</li>
<li>./dotcom-rendering/src/web/components/LiveBlogEpic.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/RewrappedComponent.tsx</li>
<li>./dotcom-rendering/src/web/components/SignInGate/dismissGate.ts</li>
<li>…and more, only first 10 files shown</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/ban-ts-comment</code> : <strong>11</strong> problems over 10 files</summary>
<ul>
<li>./dotcom-rendering/scripts/jest/setup.ts</li>
<li>./dotcom-rendering/src/amp/server/index.tsx</li>
<li>./dotcom-rendering/src/model/enhance-images.test.ts</li>
<li>./dotcom-rendering/src/model/validate.ts</li>
<li>./dotcom-rendering/src/web/browser/bootCmp/init.ts</li>
<li>./dotcom-rendering/src/web/components/ElementContainer.tsx</li>
<li>./dotcom-rendering/src/web/components/InteractiveBlockComponent.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/ReaderRevenueLinks.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx</li>
<li>./dotcom-rendering/src/web/lib/braze/buildBrazeMessages.ts</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/no-misused-promises</code> : <strong>9</strong> problems over 5 files</summary>
<ul>
<li>./dotcom-rendering/src/web/components/Callout/FileUpload.tsx</li>
<li>./dotcom-rendering/src/web/components/CalloutBlockComponent.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/InstagramBlockComponent.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/ReaderRevenueDev.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/UnsafeEmbedBlockComponent.importable.tsx</li>
</ul>
</details>

<details>
<summary><code>react/jsx-curly-newline</code> : <strong>8</strong> problems over 8 files</summary>
<ul>
<li>./dotcom-rendering/src/web/components/Branding.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/Callout/RadioSelect.tsx</li>
<li>./dotcom-rendering/src/web/components/Callout/Select.tsx</li>
<li>./dotcom-rendering/src/web/components/Callout/TextArea.tsx</li>
<li>./dotcom-rendering/src/web/components/Callout/TextInput.tsx</li>
<li>./dotcom-rendering/src/web/components/Card/components/CardBranding.tsx</li>
<li>./dotcom-rendering/src/web/components/InstagramBlockComponent.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/UnsafeEmbedBlockComponent.importable.tsx</li>
</ul>
</details>

<details>
<summary><code>no-case-declarations</code> : <strong>7</strong> problems over 4 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/lib/find-adslots.ts</li>
<li>./dotcom-rendering/src/model/enhance-embeds.ts</li>
<li>./dotcom-rendering/src/model/enhance-tweets.ts</li>
<li>./dotcom-rendering/src/web/lib/renderElement.tsx</li>
</ul>
</details>

<details>
<summary><code>eslint-comments/no-unused-disable</code> : <strong>7</strong> problems over 4 files</summary>
<ul>
<li>./dotcom-rendering/src/web/components/CalloutBlockComponent.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/ReaderRevenueLinks.importable.tsx</li>
<li>./dotcom-rendering/src/web/components/SlotBodyEnd/ReaderRevenueEpic.tsx</li>
<li>./dotcom-rendering/src/web/components/StickyBottomBanner/BrazeBanner.tsx</li>
</ul>
</details>

<details>
<summary><code>no-useless-escape</code> : <strong>6</strong> problems over 2 files</summary>
<ul>
<li>./dotcom-rendering/src/amp/lib/enhance.test.ts</li>
<li>./dotcom-rendering/src/lib/escapeData.tsx</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/default-param-last</code> : <strong>5</strong> problems over 1 files</summary>
<ul>
<li>./dotcom-rendering/src/web/lib/readerRevenueDevUtils.ts</li>
</ul>
</details>

<details>
<summary><code>@typescript-eslint/consistent-type-imports</code> : <strong>4</strong> problems over 1 files</summary>
<ul>
<li>./dotcom-rendering/index.d.ts</li>
</ul>
</details>

<details>
<summary><code>no-shadow</code> : <strong>4</strong> problems over 4 files</summary>
<ul>
<li>./dotcom-rendering/src/lib/fetchTickerData.ts</li>
<li>./dotcom-rendering/src/lib/footer-links.ts</li>
<li>./dotcom-rendering/src/web/browser/startup.ts</li>
<li>./dotcom-rendering/src/web/lib/braze/types.ts</li>
</ul>
</details>

<details>
<summary><code>sort-imports</code> : <strong>4</strong> problems over 4 files</summary>
<ul>
<li>./dotcom-rendering/src/web/layouts/ImmersiveLayout.tsx</li>
<li>./dotcom-rendering/src/web/layouts/InteractiveLayout.tsx</li>
<li>./dotcom-rendering/src/web/layouts/ShowcaseLayout.tsx</li>
<li>./dotcom-rendering/src/web/layouts/StandardLayout.tsx</li>
</ul>
</details>

### CODE REVIEW

**Comment by ioannakok** [comment]
Would an example of the expected output help in the PR description?

**Review by ioannakok — APPROVED** [review]
This is really good. A couple of non-blocking comments.

File: scripts/deno/eslint.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		(prev, { ruleId }) => {
+			const count = prev[ruleId] ?? 0;
+			prev[ruleId] = count + 1;
+			return prev;
+		},
+		{},
+	);
+
+	return {
+		path: filePath.replace(Deno.cwd(), '.'),
```
[the remark this answers is not in the document]
Comment (mxdvl) [reply scripts/deno/eslint.ts:52]: Yes, will do!

It replaces an absolute file path to a relative one. For example `/Users/max_duval/code/dotcom-rendering/dotcom-rendering/` becomes `./dotcom-rendering` if run from the repo's top folder.

[the remark this answers is not in the document]
Comment (ioannakok) [reply scripts/deno/eslint.ts:52]: Thank you!

**Pull request merged**